frontend/src/views/TE/Task.vue

```diff
@@ -208,6 +264,8 @@ export default class Task extends Mixins<TaskMixin, FormatTime>(
     feedback: ""
   };
   rules: any[] = [];
+  approve = false;
+  disapprove = false;
```

Comment:
Think of some more explanatory names for these models. I guess they indicate that the approval or disapproval is in confirmation mode?